pyproject.toml

```diff
@@ -55,6 +55,7 @@ docs = [
 ]
 lint = ["ruff", "mypy"]
 cli = ["matplotlib"]
+imas = ["imas-python[netcdf]>=2.0"]
```

Comment:
This is very minor (and maybe only for when we support more backends) but do we need the extra from imas-python or should we let the user deal with their own backends? 
I suppose as its an extra its up to them how they install imas. 

Talked myself out of it probably ignore this.